<launch>
    <arg name="output" default="screen"/>

    <node name="image_fetcher" pkg="adaptive_ctrl" type="image_fetcher.py" output="$(arg output)"/>
    <node name="image_processsor" pkg="adaptive_ctrl" type="image_processor.py" output="$(arg output)"/>

    <node name="cline_to_angle" pkg="adaptive_ctrl" type="discretising_node.py" output="$(arg output)">
        <param name="angle_pub_type" value="obv_"/>
    </node>

    <node name="tentacle_extractor" pkg="adaptive_ctrl" type="tentacle_extractor" output="$(arg output)"/>
    <node name="process_costmap" pkg="adaptive_ctrl" type="process_costmap.py" output="$(arg output)"/>

    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find adaptive_ctrl)/rviz/occ_map_v2.rviz"/>
</launch>